versioned_docs/version-1.1.0/install-kubeslice/kubeslice-cli/topology-configuration.mdx

```diff
@@ -146,4 +179,21 @@ configuration:
       chart_name: #{The name of the Worker Chart}
       version: #{The version of the chart to use. Leave blank for latest version}
       values: #(Values to be passed as --set arguments to helm install)
+    
+    # The ui_chart, prometheus_chart, and image_pull_secret parameters are required for the enterprise-demo option (enterprise-chart) only
```

Comment:
Add a full stop after `only` to complete the sentence. This comment is valid for the 1.0.0 topology configuration file also.

Reply:
done